src/frontend/Komponenter/Behandling/SettPåVent/SettPåVent.tsx

```diff
@@ -241,6 +272,27 @@ export const SettPåVent: FC<{ behandling: Behandling }> = ({ behandling }) => {
                                     onChange={(e) => settBeskrivelse(e.target.value)}
                                 />
                             )}
+                            {toggles[ToggleName.automatiskeOppgaverMotLokalkontor] &&
+                                !erBehandlingPåVent && (
+                                    <CheckboxGroup
+                                        legend="Alternative aksjoner"
+                                        onChange={settOppgaverMotLokalkontor}
+                                        size="small"
+                                    >
+                                        {aktuelleOppgaver.map((oppgave) => (
+                                            <Checkbox
+                                                key={oppgave}
+                                                value={oppgave as VurderHenvendelseOppgavetype}
+                                            >
+                                                {
+                                                    vurderHenvendelseOppgaveTilTekst[
+                                                        oppgave as VurderHenvendelseOppgavetype
+                                                    ]
+                                                }
+                                            </Checkbox>
+                                        ))}
```

Comment:
Hvis det ikke skal legges til mange andre oppgaver som kan opprettes her syns jeg det er lettere å lese om man fjerner filterfunksjonene og dropper map. Typ som dette: 

``` js
{erOvergangsstønad && <Checkbox>...}
{erOvergangsstønadEllerSkolepenger && <Checkbox>...}
```

Da slipper man også kanskje castingen av oppgave

Reply:
Gikk for Charlie sin løsning :) [61ffba8](https://github.com/navikt/familie-ef-sak-frontend/pull/2283/commits/61ffba82e3fb2ffd4985ebf93bd2044bb8d647b2)